src/oic/extension/proof_of_possesion.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, *args, **kwargs):
         # mapping from signed pop token to access token in db
         self.access_tokens: Dict[JWS, str] = {}
 
-    def token_endpoint(self, dtype="urlencoded", **kwargs):
+    def token_endpoint(self, request="", authn="", dtype="urlencoded", **kwargs):
```

Comment:
Doesn't this technically change the signature, if someone calls this with *args?

It might be better to use token_endpoint(self, dtype="urlencoded", request="", authn="", **kwargs) or mark things as keyword only arguments altogether?

Reply:
True, will try to reorder that...

Reply:
Well, no... mypy complains on the "wrong" order, so we have to keep it the way it is here.

I do hope that this piece of code is not widely used and we can bump the version to be incompatible...

Reply:
ok.